examples/snake/ws-resources.json

```
@@ -18,5 +18,6 @@
 	   "description": "This is a walrus site.",
 	   "project_url": "https://github.com/MystenLabs/walrus-sites/",
 	   "creator": "MystenLabs"
-	}
+	},
+  "name": "Walrus Snake Game"
```

Suggestion: currently the flag is `--site-name`. Use this for consistency?
```suggestion
  "site-name": "Walrus Snake Game"
```